src/cljs/metafacture_playground/events.cljs

```diff
@@ -32,15 +44,43 @@
   (let [fields [:data :flux :fix]]
     (reduce
      (fn [db field-to-empty]
-       (assoc-in db [:input-fields field-to-empty] ""))
+       (assoc-in db [:input-fields field-to-empty :content] ""))
      db
      fields)))
 
 (re-frame/reg-event-db
  :clear-all
  clear-all)
 
-;; Processing
+;;; Processing
+
+;; Fake process for developing the frontend
+
+(re-frame/reg-event-db
+ :fake-response
+ (fn [db _]
+   (-> db
+       (assoc-in [:result :loading?] false)
+       (assoc-in [:result :content] (str "<?xml version=\"1.0\" encoding=\"UTF-8\"?>"
+                                         "\n<collection>"
+                                         "\n"
+                                         "\n	<record>"
+                                         "\n		<id>1</id>"
+                                         "\n		<title>Faust</title>"
+                                         "\n		<author>Goethe</author>"
+                                         "\n	</record>"
+                                         "\n"
+                                         "\n	<record>"
+                                         "\n		<id>2</id>"
+                                         "\n		<title>Räuber</title>"
+                                         "\n		<author>Schiller</author>"
+                                         "\n	</record>"
+                                         "\n"
+                                         "\n</collection>\n")))))
+
+(defn fake-process [{:keys [db]} [_ _ _ _]]
+  {:db (assoc-in db [:result :loading?] true)
+   :fx [[:dispatch-later {:ms 4000 :dispatch [:fake-response]}]]})
 

```

Comment:
Should this output a result when running `lein watch`? Result stays empty for me when I click "Process".

Reply:
That's a really simple implementation of fake-processing so that you have to change the function in
`(re-frame/reg-event-fx
 :process
 process)`
manually to `fake-process` to make it work.
At the moment I don't use `fake-process`, maybe I should delete it again. What do you think?

Reply:
Ah, I see. I think it's good to have a way to get a result for testing when running `lein watch`. Would it be possible to detect if we're running `watch` and use `fake-process` automatically?

Reply:
I pushed bb6fad6 to use fake-process automatically when using `lein watch` 👍